docs/src/documentation/GetStarted.htm

```diff
@@ -22,6 +22,10 @@
           {'npm config set http-proxy http://\{host\}:\{port\}'}</br>
           {'npm config set https-proxy https://\{host\}:\{port\}'}
           </code></pre>
+          <p>If you find problem on downloading packages through https proxy, try using http protocol in your https-proxy varialbe, as in:</p>
```

Comment:
Can we change it to: `if you find problems`.. and also there is a typo in variable as in `varialbe`